src/tests/ftest/nvme/nvme_io_stats.py

```diff
@@ -6,18 +6,19 @@
 """
 import re
 from ior_test_base import IorTestBase
-from general_utils import get_log_file, run_task
+from general_utils import get_log_file, run_pcmd
```

Comment:
(pylint-syntax-error) Cannot import 'general_utils' due to syntax error 'invalid syntax (<unknown>, line 888)'